keel-schema-generator/src/main/kotlin/com/netflix/spinnaker/keel/schema/Schema.kt

```diff
@@ -18,11 +18,11 @@ data class RootSchema(
   val description: String?,
   val properties: Map<String, Schema>,
   val required: SortedSet<String>,
-  val discriminator: OneOf.Discriminator? = null,
+  val allOf: List<ConditionalSubschema>? = null,
   val `$defs`: SortedMap<String, Schema>
 ) {
   @Suppress("unused", "PropertyName")
-  val `$schema`: String = "https://json-schema.org/draft/2019-09/schema"
+  val `$schema`: String = "http://json-schema.org/draft/2019-09/schema#"
```

Comment:
What's the motiviation for this URL change?

Reply:
I had just gotten it wrong (I think because I copied & pasted from my URL bar). Update was based on this documentation: http://json-schema.org/understanding-json-schema/reference/schema.html#schema